### PR TITLE
Finish 8

### PR DESCRIPTION
**Pull request by sergiuHudrea**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -314,4 +314,134 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+        const vote_updates = {
+            inc_votes: -10
+        };
+        return request(app)
+            .patch('/api/reviews/2')
+            .send(vote_updates)
+            .expect(200)
+            .then(( {body} ) => {
+                const {review} = body;
+                expect(review).toBeInstanceOf(Object);
```
Comment (RichardNeat) [inline __tests__/app.test.js:330]: We don't need to explicitly check for an object here as on the next line we are saying it is an object

File: __tests__/app.test.js [hunk 2]
```diff
@@ -314,4 +314,134 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+                expect(review).toEqual({
```
Comment (RichardNeat) [inline __tests__/app.test.js:331]: Consider a more flexible jest matcher here, if our database is ever updated we don't want this test to fail and have to go back and change it

File: __tests__/app.test.js [hunk 3]
```diff
@@ -314,4 +314,134 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+        const vote_updates = {
+            inc_votes: 5
+        };
+        return request(app)
+            .patch('/api/reviews/2')
+            .send(vote_updates)
+            .expect(200)
+            .then(( {body} ) => {
+                const {review} = body;
+                expect(review).toBeInstanceOf(Object);
```
Comment (RichardNeat) [inline __tests__/app.test.js:356]: Same as above

File: __tests__/app.test.js [hunk 4]
```diff
@@ -314,4 +314,134 @@ describe("POST /api/reviews/:review_id/comments", () => {
...
+            inc_votes: 5,
+            age: 87
+        };
+        return request(app)
+            .patch('/api/reviews/2')
+            .send(vote_updates)
+            .expect(200)
+            .then(( {body} ) => {
+                const {review} = body;
+                expect(review).toBeInstanceOf(Object);
```
Comment (RichardNeat) [inline __tests__/app.test.js:383]: Same as above

File: controllers/games.js [hunk 5]
```diff
@@ -54,4 +54,18 @@ exports.postComments = (req, res, next) => {
         .catch((err) => {
             next(err);
         })
+}
+
+exports.patchReviewsById = (req, res, next) => {
```
Comment (RichardNeat) [inline controllers/games.js:59]: I'm not sure we need a checkExists in this situation, as id is a foreign key I believe we should get an SQL error if a non existent id is passed in to the patch request. Run it that way and check - see what you get.

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Hi Sergiu, great stuff just some considerations in my comments below please.
Could you also drop me a message when you see this regarding error handling for missing or incorrect body, I'm not sure where you are handling that situation currently could you have a look and let me know? Ta

File: __tests__/app.test.js (hunk 2 above)
Comment (sergiuHudrea) [reply __tests__/app.test.js:331]: I am sorry but I dont understand what you mean.

**Pull request merged**